### PR TITLE
app-misc/rdfind: Version bump 1.5.0.

### PR DESCRIPTION
**Pull request by tastytea**

### CODE REVIEW

File: app-misc/rdfind/rdfind-1.5.0.ebuild [hunk 1]
```diff
@@ -0,0 +1,24 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
```
Comment (juippis) [inline app-misc/rdfind/rdfind-1.5.0.ebuild:6]:
```suggestion
WANT_AUTOMAKE=1.15
inherit autotools
```
or even
```suggestion
WANT_AUTOMAKE=
inherit autotools
```
is enough. I see upstream requires 1.15 at minimum.

(Note that this is currently broken on EAPI-8, but it's reported and likely fixed before you read this message)

**Review by juippis — COMMENTED** [review]
I can't compile 1.5.0, 
```
>>> Emerging (1 of 1) app-misc/rdfind-1.5.0::gentoo
>>> Failed to emerge app-misc/rdfind-1.5.0, Log file:
>>>  '/var/tmp/portage/vbslogs/build/app-misc/rdfind-1.5.0:20210815-141320.log'
>>> Jobs: 0 of 1 complete, 1 failed                 Load avg: 0.30, 0.15, 0.08
 * Package:    app-misc/rdfind-1.5.0
 * Repository: gentoo
 * Maintainer: gentoo@tastytea.de proxy-maint@gentoo.org
 * Upstream:   https://github.com/pauldreik/rdfind/issues
 * USE:        abi_x86_64 amd64 elibc_glibc kernel_linux userland_GNU
 * FEATURES:   network-sandbox preserve-libs sandbox userpriv usersandbox

>>> Unpacking source...
>>> Unpacking rdfind-1.5.0.tar.gz to /var/tmp/portage/app-misc/rdfind-1.5.0/work
>>> Source unpacked in /var/tmp/portage/app-misc/rdfind-1.5.0/work
>>> Preparing source in /var/tmp/portage/app-misc/rdfind-1.5.0/work/rdfind-releases-1.5.0 ...
 * Running eautoreconf in '/var/tmp/portage/app-misc/rdfind-1.5.0/work/rdfind-releases-1.5.0' ...
 * Running 'aclocal' ...
 [ ok ]
 * Running 'autoconf --force' ...
 [ ok ]
 * Running 'autoheader' ...
 [ ok ]
 * Running 'automake --add-missing --copy --force-missing' ...
 [ ok ]
 * Running elibtoolize in: rdfind-releases-1.5.0/
>>> Source prepared.
>>> Configuring source in /var/tmp/portage/app-misc/rdfind-1.5.0/work/rdfind-releases-1.5.0 ...
 * econf: updating rdfind-releases-1.5.0/config.guess with /usr/share/gnuconfig/config.guess
 * econf: updating rdfind-releases-1.5.0/config.sub with /usr/share/gnuconfig/config.sub
./configure --prefix=/usr --build=x86_64-pc-linux-gnu --host=x86_64-pc-linux-gnu --mandir=/usr/share/man --infodir=/usr/share/info --datadir=/usr/share --sysconfdir=/etc --localstatedir=/var/lib --disable-dependency-tracking --disable-silent-rules --docdir=/usr/share/doc/rdfind-1.5.0 --htmldir=/usr/share/doc/rdfind-1.5.0/html --libdir=/usr/lib64
checking for a BSD-compatible install... /usr/lib/portage/python3.10/ebuild-helpers/xattr/install -c
checking whether build environment is sane... yes
checking for a race-free mkdir -p... /bin/mkdir -p
checking for gawk... gawk
checking whether make sets $(MAKE)... yes
checking whether make supports nested variables... yes
checking for x86_64-pc-linux-gnu-g++... x86_64-pc-linux-gnu-g++
checking whether the C++ compiler works... yes
checking for C++ compiler default output file name... a.out
checking for suffix of executables... 
checking whether we are cross compiling... no
checking for suffix of object files... o
checking whether the compiler supports GNU C++... yes
checking whether x86_64-pc-linux-gnu-g++ accepts -g... yes
checking for x86_64-pc-linux-gnu-g++ option to enable C++11 features... none needed
checking whether make supports the include directive... yes (GNU style)
checking dependency style of x86_64-pc-linux-gnu-g++... none
checking whether make sets $(MAKE)... (cached) yes
checking for stdio.h... yes
checking for stdlib.h... yes
checking for string.h... yes
checking for inttypes.h... yes
checking for stdint.h... yes
checking for strings.h... yes
checking for sys/stat.h... yes
checking for sys/types.h... yes
checking for unistd.h... yes
checking for grep that handles long lines and -e... /bin/grep
checking for egrep... /bin/grep -E
no
checking whether to enable assertions... yes
checking for nettle/sha.h... yes
checking for main in -lnettle... yes
checking for stat... yes
checking for special C compiler options needed for large files... no
checking for _FILE_OFFSET_BITS value needed for large files... no
checking for C++11 support or better... yes
checking check for fallthrough support... yes
checking that generated files are newer than configure... done
configure: creating ./config.status
config.status: creating Makefile
config.status: creating config.h
config.status: executing depfiles commands
>>> Source configured.
>>> Compiling source in /var/tmp/portage/app-misc/rdfind-1.5.0/work/rdfind-releases-1.5.0 ...
make -j16 -l10 
make  all-am
make[1]: Entering directory '/var/tmp/portage/app-misc/rdfind-1.5.0/work/rdfind-releases-1.5.0'
x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I.     -march=native -O2 -pipe -frecord-gcc-switches -c -o rdfind.o rdfind.cc
x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I.     -march=native -O2 -pipe -frecord-gcc-switches -c -o Checksum.o Checksum.cc
x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I.     -march=native -O2 -pipe -frecord-gcc-switches -c -o Dirlist.o Dirlist.cc
x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I.     -march=native -O2 -pipe -frecord-gcc-switches -c -o Fileinfo.o Fileinfo.cc
x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I.     -march=native -O2 -pipe -frecord-gcc-switches -c -o Rdutil.o Rdutil.cc
x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I.     -march=native -O2 -pipe -frecord-gcc-switches -c -o EasyRandom.o EasyRandom.cc
x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I.     -march=native -O2 -pipe -frecord-gcc-switches -c -o UndoableUnlink.o UndoableUnlink.cc
x86_64-pc-linux-gnu-g++ -DHAVE_CONFIG_H -I.     -march=native -O2 -pipe -frecord-gcc-switches -c -o CmdlineParser.o CmdlineParser.cc
rdfind.cc: In function 'Options parseOptions(Parser&)':
rdfind.cc:225:30: error: 'numeric_limits' is not a member of 'std'
  225 |     o.maximumfilesize = std::numeric_limits<decltype(o.maximumfilesize)>::max();
      |                              ^~~~~~~~~~~~~~
rdfind.cc:225:45: error: expected primary-expression before 'decltype'
  225 |     o.maximumfilesize = std::numeric_limits<decltype(o.maximumfilesize)>::max();
      |                                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~
make[1]: *** [Makefile:662: rdfind.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory '/var/tmp/portage/app-misc/rdfind-1.5.0/work/rdfind-releases-1.5.0'
make: *** [Makefile:537: all] Error 2
 * ERROR: app-misc/rdfind-1.5.0::gentoo failed (compile phase):
 *   emake failed
 * 
 * If you need support, post the output of `emerge --info '=app-misc/rdfind-1.5.0::gentoo'`,
 * the complete build log and the output of `emerge -pqv '=app-misc/rdfind-1.5.0::gentoo'`.
 * The complete build log is located at '/var/tmp/portage/vbslogs/build/app-misc/rdfind-1.5.0:20210815-141320.log'.
 * For convenience, a symlink to the build log is located at '/var/tmp/portage/app-misc/rdfind-1.5.0/temp/build.log'.
 * The ebuild environment file is located at '/var/tmp/portage/app-misc/rdfind-1.5.0/temp/environment'.
 * Working directory: '/var/tmp/portage/app-misc/rdfind-1.5.0/work/rdfind-releases-1.5.0'
 * S: '/var/tmp/portage/app-misc/rdfind-1.5.0/work/rdfind-releases-1.5.0'
```

1.4.1 works just fine. Using ~unstable, so gcc-11 etc.

File: app-misc/rdfind/rdfind-1.5.0.ebuild (hunk 1 above)
Comment (tastytea) [reply app-misc/rdfind/rdfind-1.5.0.ebuild:7]: Fixed the compile error (I think I accidentally tested the old version before 🤦) and set `WANT_AUTOMAKE`. Although I [couldn't find](https://github.com/pauldreik/rdfind/search?q=1.15) a version requirement upstream?

**Review by juippis — APPROVED** [review]
You are correct, I misread / mixed PRs with the automake requirement. I'm going to remove it while merging, thanks!

PS you should upstream that patch :)

**Pull request closed without merging**